### ISSUE TITLE
All bees producing bone comb after updating to 0.2.4.jar

### ISSUE THREAD

**Issue by demfels** [JDKDigital/productive-bees#71]
Running the updated version 0.2.4 on minecraft version 1.15.2

all bees are producing bone comb after updating to this version.  

![image](https://user-images.githubusercontent.com/4838280/90708608-d74ade80-e25f-11ea-90f8-567b013fa95a.png)

**Comment by JaisDK** [comment]
You might have some skeletal bees visiting the hive, they move in automatically if the area around hive is not well lit.

**Comment by Phayt** [comment]
I have 8 hives with 2 to 3 bees per hive and every bee (None of which are skeleton bees and all of which are well lighted) is producing bone comb

**Comment by demfels** [comment]
Right.. this isn't a situation of a skeletal bee sneaking in... All bees are producing ONLY bone comb, which means no other comb is accessible in the game.

**Comment by JaisDK** [comment]
Which modpack are you guys playing?

**Comment by demfels** [comment]
https://www.curseforge.com/minecraft/modpacks/ultimate-pack-1-15-2/files/all

**Comment by JaisDK** [comment]
I am unable to reproduce this even with the modpack, but I can only test on fresh single player worlds.

Are you having issues if you load up a fresh single player world?

also, could you try the following on the world where you are having issues:
- run the command `/reload` and see if that fixes it
- break and replace or build new beehives
- spawn fresh bees (although this is unlikely to help)

**Comment by Phayt** [comment]
Thank you, I will give that a try. I noted after a server restart that now
all of my bees are producing Lapis.

**Comment by JaisDK** [comment]
Seems it's just picked one at random, can you provide me with the server log?

**Comment by Phayt** [comment]
Dem may be able to. I am just a player, in survival, I built new hives and
caged all of my bees and replaced them in new hives. All continue to create
lapis combs

**Comment by demfels** [comment]
https://drive.google.com/file/d/1YJHk3KZny626Rd4pPcAU7bgfPaw8Jjwf/view?usp=sharing

Server Logs - thanks for checking into this!

**Comment by JaisDK** [comment]
Thanks for reporting and thanks for the logs. Unfortunately there's nothing in there to shed light on the issue.

Do you have any resourcepacks installed on the server or client?

**Comment by demfels** [comment]
No resource packs.

**Comment by demfels** [comment]
So I created a test world and confirmed as well.. This issue seems to only be happening in our world that existed previous to the patch.?  Any suggestions?  Do you think restoring the old version of the mod would do anything?
